navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/MapboxNavigation.java

```diff
@@ -47,10 +49,10 @@
   private boolean isBound;
 
   // Navigation variables
-  private CopyOnWriteArrayList<AlertLevelChangeListener> alertLevelChangeListeners;
-  private CopyOnWriteArrayList<NavigationEventListener> navigationEventListeners;
-  private CopyOnWriteArrayList<ProgressChangeListener> progressChangeListeners;
-  private CopyOnWriteArrayList<OffRouteListener> offRouteListeners;
+  private List<AlertLevelChangeListener> alertLevelChangeListeners;
```

Comment:
It seems unnecessary at least for now since we aren't passing these list to a different thread.